element_lab/export/nwb.py

```diff
@@ -36,16 +34,30 @@ def _project_to_nwb_dict(project_key: dict) -> dict:
         dict: Dictionary with NWB parameters.
     """
 
-    project_info = (lab.Project & project_key).fetch1()
-    return dict(
-        experiment_description=project_info.get("project_description"),
-        keywords=(lab.ProjectKeywords() & project_key).fetch("keyword").tolist()
-        or None,
-        related_publications=(lab.ProjectPublication() & project_key)
-        .fetch("publication")
-        .tolist()
-        or None,
-    )
+    try:  # if using project schema (not lab.Project), fetch1 will DataJointError
+        project_info = (lab.Project & project_key).fetch1()
+        return dict(
+            experiment_description=project_info.get("project_description"),
+            keywords=(lab.ProjectKeywords() & project_key).fetch("keyword").tolist()
+            or None,
+            related_publications=(lab.ProjectPublication() & project_key)
+            .fetch("publication")
+            .tolist()
+            or None,
+        )
+    except DataJointError:
+        return dict(
+            experiment_description=(project.Project & project_key).fetch1(
+                "project_title"
+            )
+            or None,
```

Comment:
```suggestion
        return dict(
            experiment_description=(project.Project & project_key).fetch1(
                "project_title"
            ),
```

Reply:
Is `or None` required?

Reply:
No, I think in the case of a `fetch1` 'or None' wouldn't do anything for us. `fetch1` will throw an error if no entry is available